packages/spaces/src/SpacesLink.js

```diff
@@ -230,6 +231,7 @@ const Link = ({
                     'mb-0': !showDescription || !description,
                     'pt-3': stacked,
                     'text-center': stacked,
+                    [titleClassName]: !!titleClassName,
```

Comment:
`classnames` accepts multiple args so i think we should use that here. It will remove any falsy values
```
classNames(<existing object>, titleClassName)
```